Require setting all inputs before execution.

shoumikhin opened the pull request:
Summary: Method never cared to check all the inputs were properly set before execution and basically used some default allocated memory for unset inputs. Here we introduce a safety check to guarantee users don't forget to set all inputs explicitly.

Differential Revision: D79849134

JacobSzwejbka reviewed (approved):
Tentively approving but please watch ci carefully since this is a breaking change

shoumikhin committed:
Require setting all inputs before execution. (#13207)

Summary:

Method never cared to check all the inputs were properly set before execution and basically used some default allocated memory for unset inputs. Here we introduce a safety check to guarantee users don't forget to set all inputs explicitly.

Reviewed By: JacobSzwejbka

Differential Revision: D79849134

shoumikhin committed:
Fix error handling macros to work well with MSVC. (#13214)

Summary:
Fix the ET_CHECK_OK_OR_RETURN_ERROR macro so MSVC picks the right overload for 1 vs 2+ arguments. This removes bogus trailing-comma cases and resolves MSVC compile errors (C2440, C2064, C2059, C4834).


Reviewed By: JacobSzwejbka

Differential Revision: D79865456

shoumikhin committed:
Merge branch 'main' into export-D79849134
